media-plugins/kodi-inputstream-adaptive/Manifest

```diff
@@ -0,0 +1 @@
+EBUILD kodi-inputstream-adaptive-9999.ebuild 492 SHA256 604c88d1cd147222a90e6f77320f9ea8481fdd4e38c2c7f86bc0a0d12aa7a837 SHA512 de66e63289a25d7ba5573197387d2550ce81f0904d88f0c11ca9bc956abba36711f81eb1595fdb53538a7365f5aa82d1b39de14ea569d5eac720b8413ec02c2d WHIRLPOOL 4718a45e77d312558b8582dc1bdf10d3382ea3a43d109772841cf1f2aff49305e9758e2c711e97329f5c3f8bb443f9d86b87c53520721490097cd707242be5a8
```

Comment:
@candrews these don't belong in the git tree. Could you see to it that all the `EBUILD` lines disappear from the Manifests?